splitgpg2/__init__.py

```diff
@@ -0,0 +1,804 @@
+#!/usr/bin/python3
+# split-gpg2.py
+# Copyright (C) 2014  HW42 <hw42@ipsumj.de>
```

Comment:
Please remove one space to make indentation consistent.

Reply:
ok